front-end/src/components/layout/Breadcrumbs.js

```diff
@@ -0,0 +1,57 @@
+import { useLocation } from "react-router-dom";
+import MUIBreadcrumbs from "@mui/material/Breadcrumbs";
+import Paper from "@mui/material/Paper";
+import Container from "@mui/system/Container";
+
+import Link from "./Link";
+
+import "./Breadcrumbs.css";
+
+function getPathFriendlyName(path) {
+  const friendlyNames = {

```

Comment:
then again, it might not make sense for it to match in every case... i don't know

Reply:
I was originally thinking this would be "private" to the breadcrumbs component but you make a good point about sharing the logic w/ the navbar!. 

I'll create a card to refactor the breadcrumbs and navbar to use the same friendly names, that way this review won't get too huge.

Reply:
Sounds good!

Reply:
Tracked in #82